lib/utils/logger/index.js

```diff
@@ -1,3 +1,19 @@
+const chalk = require('chalk');
+
+var Logger = function(){};
+
+Logger.prototype.log = function(...args) {
+  console.log(...args);
+}
+
+Logger.prototype.warn = function(...args) {
+  console.log(chalk.yellow.bold(...args));
+}
+
+Logger.prototype.err = function(...args) {
+  console.log(chalk.red.bold(...args));
+}
+
 module.exports = {
-  logger: console
-};
+  logger: new Logger()
```

Comment:
I'm not sure if we need a constructor for this. I think we should just use an object literal since we don't need to access anything in the `this` context. 

So maybe like
 ```const logger = { warn: (...args) => console.log(...args) }```

That way we don't need to `new` it.

Reply:
Hum, that's how I had it first but I didn't get it working. 
Let me try again

Reply:
That's what I get if I try to use an object literal 

```
/Users/jlengrand/IdeaProjects/mergify/lib/utils/logger/index.js:4
  log = (...args) =>{
  ^^^^^^^^^^^^^^^^^^^

SyntaxError: Invalid shorthand property initializer
```

Seems like variable arguments is not supported in the same way in there.

Reply:
Nevermind, got it working!